doc/stdenv/cross-compilation.chapter.md

```diff
@@ -123,6 +132,9 @@ software floating point emulation.  `libgcc` would be a "target→ *" dependency
 
 Some frequently encountered problems when packaging for cross-compilation should be answered here. Ideally, the information above is exhaustive, so this section cannot provide any new information, but it is ludicrous and cruel to expect everyone to spend effort working through the interaction of many features just to figure out the same answer to the same common problem. Feel free to add to this list!
 
+#### How do I test cross-compilation using emulation? {#cross-qa-emulation}
+Compile the package and run `nix-shell -p qemu`, then run `qemu-system-<arch>` where `<arch>` is the architecture of interest.  Alternatively, Nixpkgs has some logic to dispatch to the right emulator, see [\#106375](https://github.com/NixOS/nixpkgs/issues/106375)
```

Comment:
Probably better to actually document `lib.systems.emulatorFor` than to point to a github issue...